File: Modul 4 Sem 1/PHYTHON/PRAK403-2410817110005-MuhammadRizqanKarim.py
angka1 = int(input("Masukkan angka pertama: "))
angka2 = int(input("Masukkan angka kedua: "))

if angka1 > angka2:
    for i in range(angka2, angka1 + 1):
        print(f"{i} {angka1 + angka2 - i}", end=" - " if i != angka1 else "")
else:
    for i in range(angka1, angka2 + 1):
        print(f"{i} {angka1 + angka2 - i}", end=" - " if i != angka2 else "")

File: Modul 4 Sem 1/C/PRAK403-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>

int main() {
    int angka1, angka2;
    printf("Masukkan angka pertama: ");
    scanf("%d", &angka1);
    printf("Masukkan angka kedua: ");
    scanf("%d", &angka2);

    if (angka1 > angka2) {
        for (int i = angka2; i <= angka1; i++) {
            printf("%d %d", i, angka1 + angka2 - i);
            if (i != angka1) printf(" - ");
        }
    } else {
        for (int i = angka1; i <= angka2; i++) {
            printf("%d %d", i, angka1 + angka2 - i);
            if (i != angka2) printf(" - ");
        }
    }
    return 0;
}